src/plugins/sockets/inGameEvents.js

```diff
@@ -46,92 +46,36 @@ export async function handleInGameEvents(evData, newRoute = null) {
     case SocketEvent.CONCEDE:
     case SocketEvent.RE_LOGIN:
     case SocketEvent.SPECTATOR_JOINED:
-      gameStore.resetPNumIfNullThenUpdateGame(evData.game);
-      break;
-    case SocketEvent.SCUTTLE:
-      gameStore.processScuttle(evData);
-      break;
-    case SocketEvent.RESOLVE_THREE:
-      gameStore.processThrees(evData.chosenCard, evData.game);
-      break;
-    case SocketEvent.RESOLVE_FOUR:
-      gameStore.processFours(evData.discardedCards, evData.game);
-      break;
-    case SocketEvent.RESOLVE_FIVE:
-      gameStore.processFives(evData.discardedCards, evData.game);
-      break;
     case SocketEvent.RESOLVE:
     case SocketEvent.FIZZLE:
-      gameStore.resetPNumIfNullThenUpdateGame(evData.game);
-      gameStore.waitingForOpponentToCounter = false;
-      gameStore.myTurnToCounter = false;
-      if (evData.happened) {
-        switch (evData.oneOff.rank) {
-          case 3:
-            if (evData.playedBy !== gameStore.myPNum) {
-              gameStore.waitingForOpponentToPickFromScrap = true;
-            } else {
-              gameStore.pickingFromScrap = true;
-            }
-            break;
-          case 5:
-            if (evData.playedBy === gameStore.myPNum) {
-              gameStore.showResolveFive = true;
-            } else {
-              gameStore.waitingForOpponentToDiscard = true;
-            }
-            break;
-          case 4:
-            if (evData.playedBy === gameStore.myPNum) {
-              gameStore.waitingForOpponentToDiscard = true;
-            } else {
-              gameStore.showResolveFour = true;
-            }
-            break;
-          case 7:
-            if (evData.playedBy === gameStore.myPNum) {
-              gameStore.playingFromDeck = true;
-            } else {
-              gameStore.waitingForOpponentToPlayFromDeck = true;
-            }
-            break;
-          default:
-            break;
-        }
-      }
-      break;
     case SocketEvent.TARGETED_ONE_OFF:
     case SocketEvent.ONE_OFF:
     case SocketEvent.COUNTER:
-      gameStore.resetPNumIfNullThenUpdateGame(evData.game);
-      if (evData.pNum !== gameStore.myPNum) {
-        gameStore.waitingForOpponentToCounter = false;
-        gameStore.myTurnToCounter = true;
-      } else {
-        gameStore.waitingForOpponentToCounter = true;
-        gameStore.myTurnToCounter = false;
-      }
-      break;
-    // Sevens
```

Comment:
Consolidating all of these into a single case that simply updates the game was a primary motivator for this refactor. When all these variables were separate pieces of state, they each have to be set and unset at the right times abased on the specific event and who did the most recent thing. 



Now that all the modal-controls are defined as computed properties derived from the singular game phase, the possibility for desync is reduced and the handling is more consistent with fewer gotchas like "oh missed a place where X needs to be set". Generally, if a piece of data _can_ be computed, it probably should be computed so that you don't have to ensure it's properly set to the right value all the time with separate side effects